### PR TITLE
Test Expected Fee + Fix

### PR DESCRIPTION
**Pull request by fluidvanadium**

### CODE REVIEW

File: zingo-testutils/src/chain_generics/with_assertions.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ where
         .map(|(address, amount, opt_memo)| (address.as_str(), *amount, *opt_memo))
         .collect();
 
-    let proposal = from_inputs::propose(sender, raw_receivers).await.unwrap();
+    let proposal = dbg!(from_inputs::propose(sender, raw_receivers).await.unwrap());
```
Comment (zancas) [inline zingo-testutils/src/chain_generics/with_assertions.rs]: Hanging `dbg` !!

File: zingo-testutils/src/chain_generics/fixtures.rs [hunk 2]
```diff
@@ -419,16 +420,32 @@ where
...
+    //     secondary
+    //         .propose_send_all(tertiary,
+    //         get_base_address(tertiary, pool))
+    //         .await
+    //         .into_u64(),
+    //     0
+    // );
+    assert_eq!(
+        expected_fee,
+        with_assertions::propose_send_bump_sync_recipient(
```
Comment (zancas) [inline zingo-testutils/src/chain_generics/fixtures.rs:443]: Do we have any tests that exercise the `propose_send_all` logic?

File: zingo-testutils/src/fee_tables.rs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+use zcash_client_backend::PoolType;
+use zcash_client_backend::PoolType::Shielded;
+use zcash_client_backend::PoolType::Transparent;
+use zcash_client_backend::ShieldedProtocol;
+use zcash_client_backend::ShieldedProtocol::Orchard;
+use zcash_client_backend::ShieldedProtocol::Sapling;
+use zcash_primitives::transaction::fees::zip317::GRACE_ACTIONS;
+use zcash_primitives::transaction::fees::zip317::MARGINAL_FEE;
+
+/// wip
```
Comment (zancas) [inline zingo-testutils/src/fee_tables.rs]: Is this still "wip"?

Comment (zancas) [reply zingo-testutils/src/fee_tables.rs:14]: I think `one_to_one` could be `expected_one_to_one_fee` ?

Comment (zancas) [reply zingo-testutils/src/fee_tables.rs]: I love this mod!

File: zingo-testutils/src/chain_generics/fixtures.rs [hunk 4]
```diff
@@ -419,16 +420,32 @@ where
...
 
     let tertiary = environment.create_client().await;
-    with_assertions::propose_send_bump_sync_recipient(
-        &mut environment,
-        &secondary,
-        &tertiary,
-        vec![(pool, 25_000)],
-    )
-    .await;
+    let expected_fee = if make_change == 0 {
```
Comment (zancas) [inline zingo-testutils/src/chain_generics/fixtures.rs:428]: iElegante!

**Review by zancas — DISMISSED** [review]
Really nice!

File: zingo-testutils/src/chain_generics/fixtures.rs (hunk 2 above)
Comment (fluidvanadium) [reply zingo-testutils/src/chain_generics/fixtures.rs:443]: not yet

**Review by zancas — APPROVED** [review]

**Pull request merged**